js/components/operatorSignIn/expiry.tsx

```diff
@@ -136,17 +129,24 @@ export const CertificateBoxes = ({
           certifications={expiresSoon}
         />
       )}
-      {expired.length > 0 && !ignoreExpired && (
+      {certificationStatus.expired.length > 0 && !ignoreExpired && (
         <ExpiryBox
           now={now}
           mode="error"
-          title={expired.length === 1 ? "Expired card" : "Expired cards"}
+          title={
+            certificationStatus.expired.length === 1 ?
+              "Expired card"
+            : "Expired cards"
+          }
           operatorName={displayName}
-          certifications={expired}
+          certifications={certificationStatus.expired}
```

Comment:
Just a tiny style thought: What do you think about doing `const {expired, missing} = certificationStatus;` above so you can keep referring to these as `expired`.